frontend/src/scenes/settings/project/SessionRecordingSettings.tsx

```diff
@@ -206,168 +206,161 @@ export function ReplayCostControl(): JSX.Element {
+                            {
+                                label: '55%',
+                                value: '0.55',
+                            },
+                            {
+                                label: '50%',
+                                value: '0.50',
+                            },
+                            {
+                                label: '45%',
+                                value: '0.45',
+                            },
+                            {
+                                label: '40%',
+                                value: '0.40',
+                            },
+                            {
+                                label: '35%',
+                                value: '0.35',
+                            },
+                            {
+                                label: '30%',
+                                value: '0.30',
+                            },
+                            {
+                                label: '25%',
+                                value: '0.25',
+                            },
+                            {
+                                label: '20%',
+                                value: '0.20',
+                            },
+                            {
+                                label: '15%',
+                                value: '0.15',
+                            },
+                            {
+                                label: '10%',
+                                value: '0.10',
+                            },
+                            {
+                                label: '5%',
+                                value: '0.05',
+                            },
+                            {
+                                label: '0% (replay disabled)',
+                                value: '0.00',
+                            },
+                        ]}
+                        value={
+                            typeof currentTeam?.session_recording_sample_rate === 'string'
+                                ? currentTeam?.session_recording_sample_rate
+                                : '1.00'
+                        }
+                    />
+                </div>
+                <p>
+                    Use this setting to restrict the percentage of sessions that will be recorded. This is useful if you
+                    want to reduce the amount of data you collect. 100% means all sessions will be collected. 50% means
+                    roughly half of sessions will be collected.
+                </p>
+            </>
+            <>
+                <div className={'flex flex-row justify-between'}>
+                    <LemonLabel className="text-base">Minimum session duration (seconds)</LemonLabel>
+                    <LemonSelect
+                        disabledReason={!costControlFeaturesEnabled && 'Please contact support to enable this feature.'}
```

Comment:
Maybe add a link that triggers the support modal to these? To make it easier on people.

Reply:
Tooltips aren't interactive but I can do something like this.

<img width="1000" alt="Screenshot 2024-01-05 at 2 28 36 PM" src="https://github.com/PostHog/posthog/assets/21014901/01b22076-eaf4-4f34-954a-26890b1ce262">

Reply:
Hm that looks crowded with the other banner. You can add a link to a tooltip - just shove it all in the title. I was just doing this elsewhere 😄

Reply:
Tooltips disappear once you stop hovering over the target element so it's not clear how a user would interact with any content in the tooltip.

Reply:
I do agree this looks hectic.

Reply:
I feel like this looks a bit better. 
<img width="985" alt="Screenshot 2024-01-05 at 4 24 07 PM" src="https://github.com/PostHog/posthog/assets/21014901/1f2b597c-6ffc-46b2-b0ab-30e5501107ba">

Reply:
Let's move the note about required version to the "learn more in our docs" link and then we've only got one box. That version is at least two months ago, probably three, so it _shouldn't_ apply to most people that see it